tool/tctl/common/auth_command.go

```diff
@@ -713,8 +713,7 @@ func (a *AuthCommand) generateUserKeys(ctx context.Context, clusterAPI auth.Clie
 		return nil
 	}
 	if reqExpiry.Sub(expires) > time.Minute {
-		log.Warnf("Requested TTL of %s was not granted. User may have a role with a shorter max session TTL"+
-			" or an existing session ending before the requested TTL. Proceeding with %s",
+		return trace.Errorf(`Error: the credential was not issued because the requested TTL exceeded the maximum allowable value of MAX_ALLOWABLE_TTL. To successfully request a credential, please replace requested TTL %s with MAX_ALLOWABLE_TTL of %s`,
```

Comment:
```suggestion
		return trace.Errorf(`the credential was not issued because the requested TTL exceeded the maximum allowable value of MAX_ALLOWABLE_TTL. To successfully request a credential, please replace requested TTL %s with MAX_ALLOWABLE_TTL of %s`,
```

1. You can remove the leading `Error: ` prefix
2. Replace `MAX_ALLOWABLE_TTL` with the actual value.

Reply:
fixed on [94b2cda](https://github.com/gravitational/teleport/pull/15384/commits/94b2cda89ce8b4b103e961afbdddea5afa5f21c8) to remove `Error: ` prerfix & replaced `MAX_ALLOWABLE_TTL` to required value